src/main/target/ALIENFLIGHTF3/initialisation.c

```diff
@@ -43,6 +43,6 @@ void targetBusInit(void)
     #endif
 
     #ifdef USE_I2C
-        i2cInit(I2C_DEVICE);
+        i2cInitAll();
```

Comment:
Call    `i2cInitAll(i2cPinConfig());` - will need to include `include "config/config_master.h"`